### PR TITLE
marketplace: Rewrite to add Azure instructions

### PR DESCRIPTION
**Pull request by jordigh**

### CODE REVIEW

File: help/en/docs/install/grist-builder-edition.md [hunk 1]
```diff
@@ -1,28 +1,48 @@
...
 
-[Grist Builder Edition](https://aws.amazon.com/marketplace/pp/prodview-tew3ygop5xxy4) on the AWS Marketplace has everything you need to run a self-hosted Grist instance with minimal setup. Below are the initial configuration steps.
+* [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-tew3ygop5xxy4)
+* [Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/grist.grist-builder-edition)
 
-### How to log in to the Grist instance
+Once you select an offering, follow the usual procedure for each cloud provider for starting a virtual machine. You will need to allow SSH access for the virtual machine, as well as HTTP and HTTPS access. This corresponds to TCP network ports 22, 80, and 443 respectively. 
 
-During deployment, you should have been asked about creating or using key pairs. You can use this pair to log in via SSH from your terminal/bash. The default user for the Grist EC2 instance is named “ubuntu”, and you can log in to `ubuntu@[ec2-instance-public-ip]`.
+### How to log in to the Grist instance
```
Comment (nbush) [inline help/en/docs/install/grist-builder-edition.md:15]: log into

File: help/en/docs/install/grist-builder-edition.md [hunk 2]
```diff
@@ -1,28 +1,48 @@
...
 * Linux: [https://www.ssh.com/academy/ssh/command](https://www.ssh.com/academy/ssh/command)
 * macOS: [https://www.servermania.com/kb/articles/ssh-mac](https://www.servermania.com/kb/articles/ssh-mac)
 
+Once you log in, follow the instructions displayed in the console.
+
+If you need some help with the initial configuration, read on below for some specific instructions for each cloud provider.
+
+#### AWS
+
+During deployment, you should have been asked about creating or using key pairs. You can use this pair to log in via SSH from yourf terminal/bash. The default user for the Grist EC2 instance is named “ubuntu”, and you can log in to `ubuntu@[ec2-instance-public-ip]`.
```
Comment (nbush) [inline help/en/docs/install/grist-builder-edition.md:29]: yourf & log into

File: help/en/docs/install/grist-builder-edition.md (hunk 1 above)
Comment (jordigh) [reply help/en/docs/install/grist-builder-edition.md:15]: Not login to? :P

Okay, will fix.

File: help/en/docs/install/grist-builder-edition.md [hunk 3]
```diff
@@ -1,28 +1,48 @@
...
+**Note:** You need to use the `*.pem` file you received while generating key pairs on AWS. 
+
 If you don’t want to connect via SSH, AWS provides the option to connect from within the AWS console using the “Connect” button:
 
 ![AWS Connect screenshot](../images/aws-connect.png)
 
-Once you log in, follow the instructions displayed in the console.
+#### Azure
+
+When deploying the virtual machine you may be asked to use SSH keys or password, as well as the default user to log in as. The default user is named `azureuser`, and you can login as `azureuser@[azure-vm-public-ip]`.
```
Comment (nbush) [inline help/en/docs/install/grist-builder-edition.md:39]: username for AWS is in quotes, but this one is backticks. let's go with backticks for both to match the emails

**Review by nbush — CHANGES_REQUESTED** [review]
a few tiny changes, otherwise good to go!

**Comment by jordigh** [comment]
Changes done!

**Pull request merged**